modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -987,6 +987,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-156320 - Assert an Action with an Expression can be triggered after adding an entry, deleting an entry, and updating an entry."
+	@priority = "4"
+	test CanTriggerActionWithExpression {
+		task ("Given an object with a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
```

Comment:
Minor, but change object names to use the ticket number. EX. Custom Object 156320